src/main/resources/db/migration/V0004__alter_individual_request.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE INDIVIDUAL_REQUESTS ADD COLUMN ERROR_LOG_ID NUMERIC;
```

Comment:
Does this need a foreign key constraint with the error_log_id column on the error_logs table?

Reply:
Do we? We already have el_individual_request_id ON error_logs (individual_request_id)